sphinx/writers/html5.py

```diff
@@ -872,7 +872,9 @@ def visit_table(self, node: Element) -> None:
         self._table_row_indices.append(0)
 
         atts = {}
-        classes = [cls.strip(' \t\n') for cls in self.settings.table_style.split(',')]
+        classes: list[str] = [
```

Comment:
Is it required because `settings.table_sync` has type Any? Does `reveal_type(classes)` for mypy is `list[Any]`?